src/transforms/mapperTransforms.ts

```diff
@@ -251,10 +251,8 @@ function getXmlMetadata(
 }
 
 // TODO: Make sure this is the correct way to handle additionalProperties
```

Comment:
Is this comment necessary anymore?

Reply:
I'm assuming not now that all the tests pass! Removed.